test/gst-va/decode/av1.py

```diff
@@ -0,0 +1,42 @@
+###
+### Copyright (C) 2021 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ....lib.gstreamer.decoderbase import BaseDecoderTest
+from ....lib.gstreamer.util import have_gst_element
+
+spec = load_test_spec("av1", "decode", "8bit")
+
+@slash.requires(*platform.have_caps("decode", "av1_8"))
+@slash.requires(*have_gst_element("vaav1dec"))
+class DecoderTest(BaseDecoderTest):
+  def before(self):
+    super().before()
+    os.environ["GST_VAAPI_DRM_DEVICE"] = get_media().render_device
+
+  def map_formatu(self):
+    return mapformatu(self.format)
```

Comment:
This class should be moved into lib/gstreamer/va/decoder.py since all of gst-va/decode tests will be subclasses of it.  You will also need a lib/gstreamer/gst-va/util.py file with gstva-specific implementations for mapxxx, etc. functions.